Rakefile

```diff
@@ -20,13 +20,20 @@ namespace :test do
   task ios_example: :prepare do
     run_build('cauli-ios-example', 'iphonesimulator', 'iPhone 6', '12.2')
   end
+
+  desc 'Builds the project with the Swift Package Manager'
+  task spm: :prepare do
+    sh("swift build") rescue nil
+    package_manager_failed('Swift Package Manager') unless $?.success?
+  end
 end
 
 
 desc 'Run the Cauli tests'
 task :test do
   Rake::Task['test:ios'].invoke
   Rake::Task['test:ios_example'].invoke
+  Rake::Task['package_manager:spm'].invoke
```

Comment:
The current build on CI here is due to a copy-past issue 😊

In the Matomo repository there is a `package_manager` namespace where both the SPM and the Carthage build is tested. Here, the `package_manager` namespace isn't defined and thus `package_manager:spm` is not defined and can't be executed. I think there are two options:
1. Adding a `package_manager` namespace or
2. Changing this line to `Rake::Task['test:spm'].invoke`

I am fine with either. And then we can have a look if just `swift build` is sufficient or if we need to define the sdk and target.

Reply:
Thanks
I did not work with Rakefile before and did not know much about the ruby syntax here :).

I will go with the first option
>Adding a package_manager namespace or
I find it a good solution to have a separation of the task from the main test.

Reply:
I did the refactor in this commit 1c37192

Reply:
Nice!